plugins/org.jkiss.dbeaver.ext.iotdb/plugin.xml

```diff
@@ -0,0 +1,256 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?eclipse version="3.2"?>
+<plugin>
+   <extension point="org.jkiss.dbeaver.dataSourceProvider">
+      <datasource class="org.jkiss.dbeaver.ext.iotdb.IoTDBDataSourceProvider"
+                  id="iotdb"
+                  description="Generic IoTDB JDBC connector"
+                  icon="#database_icon_default"
+                  parent="generic"
+                  dialect="generic"
+                  label="Generic">
+         <tree icon="icons/iotdb_icon.png" label="IoTDB data source" path="iotdb">
```

Comment:
It's not a good idea to copy paste entire metadata tree here. Besides other issues, we will lose all localizations we already have for generic drivers.
Please try using `treeInjection` here. See examples in db2i, gaussdb, hive, gbase8s, etc